src/reuse/spdx.py

```diff
@@ -22,10 +23,47 @@ def add_arguments(parser) -> None:
     parser.add_argument(
         "--output", "-o", dest="file", action="store", type=PathType("w")
     )
+    parser.add_argument(
+        "--add-license-concluded",
+        action="store_true",
+        help=_(
+            "Populate the LicenseConcluded field. Note that REUSE cannot "
```

Comment:
Reuse should be lowercase instead of capitalized I guess as we're talking about the tool, not the spec?
```suggestion
            "Populate the LicenseConcluded field. Note that reuse cannot "
```